content/en/blog/2022/centralized-east-west-gateway/index.md

```diff
@@ -0,0 +1,295 @@
+2. Fine-grained control of traffic behavior with rich routing rules, retries, failovers, and fault injection
+3. A pluggable policy layer and configuration API supporting access controls, rate limits and quotas
+
+
+The features below are provided, but with limitations compared to the original sidecar:
+1. Secure communication in a cluster with server-side TLS encryption, strong identity-based server-side authentication and authorization.
+2. Automatic metrics, logs, and traces for all traffic going through the gateway.
+
+
+The gateway can be implemented by envoy、nginx、or commercial load balancers provided by cloud providers.  To save the resources further, the sidecar can be deployed together with ingress
+gateway.
+
+In this document, we demonstrate how to create a centralized east-west traffic gateway with Envoy:
+
+1. First, create an InternalGateway service, which deploys envoys using router mode.
+2. Create a Gateway target: Associate  the InternalGateway with the selector. The listening port can be configured through Service target.
+3. Create a VirtualService target: associate the Gateway and Service created in the last step.
+4. Configure DNS: for all the traffic targeting a service, redirect the traffic to the InternalGateway service. 
+
+Examples are demonstrated in the next section.
+
+## Related works
+
+There are a few proposals in the istio community to address those problems mentioned above. For example, (sidecars can be installed on each node instead of each pod)[1]. This way, the number of sidecars used can be reduced dramatically, and the delay is also reduced because the requests only need to go through one sidecar. However, this approach needs modifications on the node, and it can also cause resource competition between application and sidecars. Moreover, the upgrading and maintenance for the sidecars can be as demanding as the per-pod sidecar mode. 
+
+A recently released document from Istio (proposes a similar centralized sidecar idea)[2]. Since the sidecars are moved out of the nodes, it faces the similar problems we mentioned above: how to accomplish mtls and tracing features that are covered by the local sidecars. The solution used is to install an agent on the node. By this way, it provides comparable features to the original sidecars. However, because an agent is involved, complexity is introduced when the agent is installed or upgraded. It's a great solution for applications where those features are needed. For the HTTP applications that do not require mtls and end-to-end tracing, we still recommend our solution, where no modification on node is required.
+
+## Configuration examples
+
+
+Next, we introduce how to deploy centralized east-west traffic gateway.
+
+Prerequisite:
+
+1. Only one Gateway can be attached to InternalGateway.
+2.  The VirtualService managed by InternalGateway must be attached to the Gateway mention in 1.
+3. CoreDNS is required.
```

Comment:
What components and versions of Istio are required?

Reply:
Traffic hijacking is based on coredns, so coredns is a required component, and the rest of the conditions are consistent with istio itself. The version of istio is only adapted to version 1.15, but all versions should be applicable.